### PR TITLE
enhance: display disconnect result for pouch network disconnect

### PR DESCRIPTION
**Pull request by shaloulcy**
Signed-off-by: Eric Li <lcy041536@gmail.com>

### Ⅰ. Describe what this PR did

enhance: display disconnect result for pouch network disconnect

Now If we disconnect a container from a network, it will display nothing. This PR will print the success message like

```
container test is disconnected from network bridge
```

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Describe how you did it

NONE

### Ⅳ. Describe how to verify it
if **pouch network disconnect** successfully, it will print success

### CODE REVIEW

File: cli/network.go [hunk 1]
```diff
@@ -497,13 +495,26 @@ func (nd *NetworkDisconnectCommand) addFlags() {
...
+	err := apiClient.NetworkDisconnect(ctx, network, container, nd.force)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("container %s is disconnected from network %s\n", container, network)
+
+	return nil
 }
 
 // networkDisconnectExample shows examples in 'disconnect' command, and is used in auto-generated cli docs.
```
Comment (rudyfly) [inline cli/network.go]: need change the example command.

**Comment by rudyfly** [comment]
Can you add more information about this pr? Such as change format from "A" to "B"...

File: cli/network.go (hunk 1 above)
Comment (shaloulcy) [reply cli/network.go]: fine. I have updated it. PTAL, thanks

File: cli/network.go [hunk 2]
```diff
@@ -497,17 +495,30 @@ func (nd *NetworkDisconnectCommand) addFlags() {
...
 
 	ctx := context.Background()
 	apiClient := nd.cli.Client()
 
-	return apiClient.NetworkDisconnect(ctx, nd.network, nd.container, nd.force)
+	err := apiClient.NetworkDisconnect(ctx, network, container, nd.force)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("container %s is disconnected from network %s\n", container, network)
```
Comment (HusterWan) [inline cli/network.go]: how about `successfully disconnected ` ? :)

**Comment by HusterWan** [comment]
lgtm

**Pull request merged**